src/Components/PeriodicTable.js

```diff
@@ -29,15 +45,15 @@ import data from "../PeriodicTableJSON.json";
               backgroundColor: colorMap[element.category],
             }}
           >
-            <strong>{element.symbol}</strong>
-            <text className="number">{element.number}</text>
-            <text className="name">{element.name}</text>
-          </div>
+            <span className="symbol">{element.symbol}</span>
+            <span className="number">{element.number}</span>
+            <span className="name">{element.name}</span>
+          </button>
         ))}
       </div>
-    );
-  };
+      <Modal open={isOpen} onClose={() => setIsOpen(false)} content={infos} />
```

Comment:
As `isOpen` is a boolean, I would recommend renaming the prop to make that directly visible (same as you have done with the variable name):
```suggestion
      <Modal isOpen={isOpen} onClose={() => setIsOpen(false)} content={infos} />
```